bodhi/server/bugs.py

```diff
@@ -165,7 +165,8 @@ def comment(self, bug_id, comment):
 
     def on_qa(self, bug_id, comment):
         """
-        Change the status of this bug to ON_QA if it is not already ON_QA, VERIFIED, or CLOSED.
+        Change the status of this bug to ON_QA if bug.product is in admitted products list
+        and if it is not already ON_QA, VERIFIED, or CLOSED.
```

Comment:
Bodhi uses [pydocstyle](http://www.pydocstyle.org) to enforce docblock style, and one of the rules it enforces is that the first paragraph of a docblock must be a single sentence.

```
f27	bodhi/server/bugs.py:166 in public method `on_qa`:
f27	        D205: 1 blank line required between summary line and description (found 0)
f27	bodhi/server/bugs.py:166 in public method `on_qa`:
f27	        D400: First line should end with a period (not 't')
```

To fix this, I suggest mentioning the product list restriction in its own paragraph. Perhaps something like this instead:

```
Change the status of this bug to ON_QA if it is not already ON_QA, VERIFIED, or CLOSED.

This method will only operate on bugs that are associated with products listed in the bz_products setting.
```